src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,6 @@
+$watch-size: 500px;
```

Comment:
Consider adding a block comment at the beginning of the file explaining the purpose of the variables and their usage in the project 📝